src/runtime/storageNG/handle.ts

```diff
@@ -105,11 +105,12 @@ export abstract class Handle<T extends CRDTTypeRecord> {
   }
 
   abstract onUpdate(updates: T['operation'][]): void;
-  // TODO: this shuld be async and return Promise<void>.
   abstract onSync(): void;
   
   async onDesync(): Promise<void> {
-    await this.particle.callOnHandleDesync(this, e => this.reportUserExceptionInHost(e, this.particle, 'onHandleUpdate'));
+    await this.particle.callOnHandleDesync(
+        this,
+        e => this.reportUserExceptionInHost(e, this.particle, 'onHandleDesync'));
```

Comment:
is the layout change here deliberate or accidental?

Reply:
the line had the method name wrong.
the formatting is also intentional, let me know if it does not look ok (do we have a line length?)